nasa-gibs/worldview
# WV-2622 invisible flags

## Description
There are imagery layers which contain transparent pixel values which could be rendered as opaque in Worldview. It would be good to allow users to toggle these pixel values on as needed. We would want these classifications off by default, but a user could toggle them on if wanted.

One example is the [MODIS sea ice product ](http://localhost:3000/?v=-315.0774629899484,-247.99306208393455,345.39659883674705,200.86311393833853&l=MODIS_Terra_L3_Sea_Ice_Daily(disabled=0-1-2-3-4-5-6-8-9-10),VIIRS_SNPP_CorrectedReflectance_TrueColor&lg=false&t=2023-05-01-T12%3A38%3A10Z)having embedded flags for quantities such as "no data due to cloud cover". Some users may want to know why there is no data available and this would allow them to see the reason.

Fixes # WV-2622.

## How To Test

- git checkout wv-2622-invisible-flags
- npm run build
- npm run watch
- View layer with hidden data in the served tiles (such as the sea ice layer above)
- click on 'View Options' for that layer
- Toggle colors on/off & confirm the visuals update as expected

@nasa-gibs/worldview
